### DISCUSSION TITLE
有人知道刀削面是什么吗？

### DISCUSSION THREAD

**Discussion by WEssntssN** [share121/inter-knot#2993] [其他]
最近在红烧空洞调查时遇到了一个棘手的问题
![1-230211155K1503](https://github.com/user-attachments/assets/a6e824cb-218e-4f44-9f02-8f91dd431296)

**Comment by lllliufeng** [comment]
你不知道吗?!就是一只手拿着面团，另一只手里拿刀，站在开水锅前，把面团削成细长的薄片下进锅里煮的那种面!

**Comment by 502awa** [comment]
当然是刀削的面啊awa

**Comment by RaphaelHider** [comment]
是把面和调料混合，揉成面团整个下锅煮熟，捞出放到盘子上，需要左手拿叉右手拿刀，削下来一块吃一块，由于吃法十分优雅，被称为面食界的牛排。我记得十一分街就有一家刀削面馆，生意挺好的，你可以去尝尝。<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by SeeleGit** [comment]
知らないの？ ！ 片方の手に生地を持ち、もう片方の手にナイフを持って、お湯鍋の前に立って、生地を細長い薄切りにして鍋に入れて煮るタイプの麺です！
